src/hubverse_infrastructure/hubs/hubs.yaml

```diff
@@ -2,9 +2,6 @@
 # adopt Pulumi, the next step would be reading this information from each hub's admin.json config.
 # hub = admin.json: cloud.storage
 hubs:
-- hub: hubverse-infrastructure-test
```

Comment:
Probably should have been its own PR, but cleaning up some old testing resources that I'm tired of seeing